exercicios/para-casa/biblioteca.py

```diff
@@ -0,0 +1,31 @@
+from livro import Livro
+
+class Biblioteca:
+    def __init__(self):
+        self.livro = []
+        
+    def add_livro(self, livro: Livro):
+        if (isinstance(livro,Livro) == False):
+            raise TypeError(f"Esperado Livro obtido valor {livro} do tipo{type(livro)}")
+        self.livro.append(livro)
+    
+    def exibir_livros(self):
+        if len(self.livro) == 0: #verifica se a list esta vzia
+         raise TypeError("Lista Vazia") 
```

Comment:
[Apenas uma sugestão] Que acha de no lugar de TypeError usar o [IndexError](https://docs.python.org/3/library/exceptions.html#IndexError)

```
"O que é um IndexError em Python?
IndexError é uma exceção em python que ocorre quando tentamos acessar um elemento de uma lista ou tupla de um índice que não está presente na lista."
```

[Python For Beginners](https://www.pythonforbeginners.com/basics/indexerror-in-python#:~:text=Try%2DExcept%20Blocks-,What%20is%20an%20IndexError%20in%20Python%3F,is%20from%200%20to%209.)